variables.tf

```
@@ -244,6 +244,12 @@ variable "ignore_changes_task_definition" {
   default     = true
 }
 
+variable "ignore_desired_count" {
```

For consistency with `var.ignore_changes_task_definition`

```suggestion
variable "ignore_changes_desired_count" {
```